analyses/independent-samples/README.md

```diff
@@ -21,14 +23,18 @@ Komal Rathi, Run Jin, Yuanchao Zhang
 ├── 02-generate-independent-rnaseq.R
 ├── 03-qc-independent-samples.Rmd
 ├── 03-qc-independent-samples.nb.html
+├── 04-generate-independent-specimens-rnaseq-release.R
 ├── README.md
 ├── results
 │   ├── independent-specimens.rnaseq.primary-plus.eachcohort.tsv
 │   ├── independent-specimens.rnaseq.primary-plus.tsv
+│   ├── independent-specimens.rnaseq.primary-plus-release.tsv
 │   ├── independent-specimens.rnaseq.primary.eachcohort.tsv
 │   ├── independent-specimens.rnaseq.primary.tsv
+│   ├── independent-specimens.rnaseq.primary-release.tsv
 │   ├── independent-specimens.rnaseq.relapse.eachcohort.tsv
 │   ├── independent-specimens.rnaseq.relapse.tsv
+│   ├── independent-specimens.rnaseq.relapse-release.tsv
```

Comment:
Just a thought - would it be better to use `pre-release` to indicate they are not ready for release yet?